sunday-morning/resolveScopedStyles.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import * as React from "react";
+
+export default function resolveScopedStyles(scope: React.Element<any>) {
```

Comment:
does this function have some purpose here?

Reply:
I guess it's just a copy from Saturday. Removed